src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -39,13 +39,13 @@ export const Configurator = ({
 }) => {
   const queryService = getQueryService();
   const queryString = queryService.queryString;
-  const languageService = queryService.queryString.getLanguageService();
+  const languageService = queryService.queryString?.getLanguageService();
```

Comment:
Can we not mock languageService over introducing optional chaining throughout this file?

Reply:
addressed